### DISCUSSION TITLE
<class'sqlalchemy.engine.row.Row'>How can the object json.dumps()

### DISCUSSION THREAD

**Discussion by jakehu** [sqlalchemy/sqlalchemy#6946] [Usage Questions]
ERROR
```
Object of type Row is not JSON serializable
```

**Comment by CaselIT** [comment]
Hi,

You can specify a custom JSONEncoder.
Refer to https://docs.python.org/3/library/json.html . It has an example under `Extending JSONEncoder:`

**Comment by zzzeek** [comment]
you can also turn the row into a tuple or dictionary:

```py
tuple_row = tuple(row)
dict_row = dict(row._mapping)
```
